apiv2/api/openapi/openapi.yaml

```diff
@@ -5486,8 +5486,12 @@ components:
           type: array
           items:
             type: string
-            pattern: ^$|^[a-zA-Z-_0-9./:;=@?!#,<>*+()"\ \n]+$
+            pattern: |-
+              ^$|^[a-zA-Z-_0-9./:;=@?!#,<>*+()" \
+              ]+$
```

Comment:
Fix the \n to avoid a breaking line here, it can be done by adding more \ before the \n. Check the other fields.